<!-- -->
<launch>

  <machine name="local" user="$(env USER)" address="localhost"/>

  <arg name="machine" default="local" doc="Name of the machine on which the nodes should be started"/>

  <node pkg="youbot_base_driver" type="youbot_base_driver_node" name="youbot_base_driver" machine="$(arg machine)">
    <param name="publish_tf"       value="true"/>

    <param name="loop_rate"        value="100.0"/>

    <param name="parent_frame_id"  value="odom"/>
    <param name="child_frame_id"   value="odom_robot"/>

    <!--<param name="config_file_path"   value="$(find youbot_driver)/config"/>-->
    <param name="config_file_path" value="/home/youbot/robor_ws/src/youbot_driver/config"/>
    <param name="config_file_name" value="youbot-base"/>
  </node>

</launch>
<!-- -->
